src/lib.rs

```diff
@@ -151,6 +151,12 @@ pub fn packages(manager: &str) -> io::Result<String> {
                 .output()?;
             Ok(format!("{}", packages::count(output)))
         }
+        "flatpak" => {
+            let output = Command::new("flatpak")
+                .args(&["list"])
```

Comment:
Only needs to be `.arg("list")` since it's one argument.
Besides that though, looks good. Thank you for the PR. :)